components/script/dom/htmlmediaelement.rs

```diff
@@ -170,7 +170,8 @@ pub(crate) struct MediaFrameRenderer {
     old_frame: Option<ImageKey>,
     very_old_frame: Option<ImageKey>,
     current_frame_holder: Option<FrameHolder>,
-    show_poster: bool,
+    // <https://html.spec.whatwg.org/multipage/#poster-frame>
```

Comment:
Nit: This should likely be rustdoc so that it shows up on hover when using the IDE.

```suggestion
    /// <https://html.spec.whatwg.org/multipage/#poster-frame>
```